src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -249,9 +250,9 @@ class ProxyService(proxyConfig: ProxyConfig,
   /**
     * Gets the notebook server hostname from the database given a google project and cluster name.
     */
-  protected def getTargetHost(googleProject: GoogleProject, clusterName: ClusterName): Future[GetClusterResponse] = {
+  protected def getTargetHost(googleProject: GoogleProject, clusterName: ClusterName): Future[HostStatus] = {
     implicit val timeout: Timeout = Timeout(5 seconds)
-    (clusterDnsCache ? GetByProjectAndName(googleProject, clusterName)).mapTo[GetClusterResponse]
+    clusterDnsCache.projectNameToHost(DnsCacheKey(googleProject, clusterName)).mapTo[HostStatus]
```

Comment:
Getting the compilation warning
```
[warn] /Users/kyuksel/github/leonardo/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala:255:21: method apply in trait LoadingCache is deprecated: see corresponding Javadoc for more information.
[warn]     clusterDnsCache.projectNameToHost(DnsCacheKey(googleProject, clusterName)).mapTo[HostStatus]
[warn]                     ^
```
which can be fixed by using `get` instead of `apply` (which is called implicitly with the current code):
```scala
clusterDnsCache.projectNameToHost.get(DnsCacheKey(googleProject, clusterName)).mapTo[HostStatus]
```